pkg/datapath/loader/template.go

```diff
@@ -26,6 +26,7 @@ const (
 	templateSecurityID          = identity.ReservedIdentityWorld
 	templateLxcID               = uint16(65535)
 	templatePolicyVerdictFilter = uint32(0xffff)
+	templateIfIndex             = 0
```

Comment:
These constants must be non-zero IIRC? Otherwise the compiler does weird things. Maybe use math.MaxUint32?